evap/staff/views.py

```diff
@@ -275,6 +275,24 @@ def semester_lottery(request, semester_id):
     template_data =dict(semester=semester, form=form, eligible=eligible, winners=winners)
     return render(request, "staff_semester_lottery.html", template_data)
 
+@staff_required
+def semester_todo(request, semester_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+
+    responsibles = set()
+    prepared_courses = semester.course_set.filter(state__in=['prepared']).all()
+    courses = semester.course_set.filter(state__in=['prepared', 'lecturerApproved']).all()
+
+    for course in prepared_courses:
+        responsibles.add(course.responsible_contributor)
+
+    responsibles = list(responsibles)
+    responsibles.sort(key = lambda responsible: (responsible.last_name, responsible.first_name))
+
+    responsible_list = [dict(responsible=responsible, courses=[course for course in courses if course.responsible_contributor.id == responsible.id], delegates=responsible.delegates.all()) for responsible in responsibles]
+
+    template_data = dict(semester=semester, responsible_list=responsible_list)
+    return render(request, "staff_semester_todo.html", template_data)
```

Comment:
it is intended. approved courses are not necessary for the todo, so they are usually not shown. but i need to know who approved these courses if the responsible lecturer has some not approved courses left to know to whom i have to talk.